q2_bowtie2/plugin_setup.py

```diff
@@ -51,10 +52,16 @@
         ("aligned_reads", SampleData[SequencesWithQuality]),  # type: ignore
         ("unaligned_reads", SampleData[SequencesWithQuality]),  # type: ignore
         ("bowtie2_alignment", SampleData[AlignmentMap]),  # type: ignore
+        ("read_features", FeatureTable[Frequency]),  # type: ignore
```

Comment:
I think this should be some kind of SampleData semantic type, I'm not positive though

Reply:
I have researched a bit and found this thread where they talk a bit about feature tables, [https://forum.qiime2.org/t/some-confusion-about-the-feature-table-of-qiime2/13665](https://forum.qiime2.org/t/some-confusion-about-the-feature-table-of-qiime2/13665). From this thread it could make sense to use FeatureTable. 
I have also had a look at SampleData. The only one that is a bit similar is the SampleData[AlphaDiversity], but here they are only outputting a vector. Where we would like to output multiple columns. Using SampleData also requires that we define a format and type for ReadFeatures.